Ada_Zangemann-cs.txt

```diff
@@ -186,784 +186,712 @@ Society should support those who respect others' freedom.
-left of the roof supports a clothesline on which red clothespins hold a 
-green tank top, a pair of light blue pants, and white underpants. To the 
-right of the house there is a veranda on which there is installed a 
-solar panel. Under the veranda there are a mattress, a red oil drum, and 
-a gray case.
-Behind the house there are piles of landfill objects among which you can 
-see a washing machine, a car, and various wooden boards. 
+OBRÁZEK: V popředí odchází Ada z domu do školy, v pozadí je její dům a
+vrakoviště. Ada má hnědý batoh se žlutými popruhy.
+Nalevo od Ady je červené auto, Volkswagen Brouk, bez kol a jenom s jedním
+světlometem, který už visí jenom na drátech.
+Nalevo před autem leží dvě pneumatiky a dvě krabice. Jedna z nich je
+dřevěná s převráceným kbelíkem, prázdnou plechovkou, mezi nějakými rostlinami
+se žlutými květy.
+V pozadí je Adina chatrč a maminka s bratrem, kteří ji mávají na rozloučenou.
+Na zemi před nimi sedí pes.
+Alan sedí na dvou schodcích do domku. Má krátké tmavé vlasy a je oblečený
+do bílého trička s krátkými rukávy olemovanými červeným lemem a dvěma červenými
+pásky na spodním okraji trička. Alan má modré kraťasy a je naboso.
+Adina máma má tmavou rtěnku a černé vlasy svázané bílou stuhou. Má krátké
+modré tričko s bílým zakončením, bílé kalhoty a žluté boty.
+Adina chatrč je dřevěná s bílou přední stěnou. Střecha je z tmavě červených
+vlnitých plechů. Vpředu jsou kromě dveří ještě dvě okna a další okno je na
+levé straně. Na střeše stojí žlutá kachna. Střechou se prodírá bílý komín.
+Na anténě na levé straně střechy je uvázaná šňůra na prádlo, na které visí
+zelený top, světle modré kalhoty a bílé kalhotky, přichycené červenými kolíčky na prádlo.
+Napravo od domu je veranda se solárním panelem. Pod verandou je matrace, červený sud od
+oleje a šedý kufr.
+Za domem jsou hromady starých věcí, na kterých můžeme rozeznat pračku, auto,
+a mnoho dřevěných prken.
 ]]
 
 ** 5 **
 
 [[I:ada-p05
-IMAGE: Prestižní časopisy (“HVĚZDY“, “PENÍZE”, “Richest Man”) na kterých
+IMAGE: Prestižní časopisy (“Hvězdy“, “Peníze”, “Největší boháč”) na kterých
 je Zangemann a jeho vila.
 ]]
-Daleko, na druhém konci města, žil slavný vynálezce jménem Zangemann.  Byl
-nesmírně bohatý.Žádný papír na světě nebyl dostatečně velký, aby na něj šlo
-napsat kolik má peněz. Tolik čísel a nul! Žil v obrovském domě s bazénem a
-tobogánem, spoustou schodů a věží, stovky oken a tolik místností, že on sám se
-v nich často ztrácel. Zangemannovo sídlo stálo vysoko na kopeci. Odkud se mohl
-dívat na celé město.
+Daleko odtud, na opačném konci, žil slavný vynálezce Zangemann. Byl nesmírně bohatý. Žádný list papíru na světě nebyl dost velký na to, aby se na něj vešel zůstatek jeho účtu se všemi těmi číslicemi a nulami. Bydlel v obrovském domě s bazénem a tobogánem, mnoha schodišti, věžemi, stovkami oken a tolika pokoji, že i on sám se v nich pravidelně ztrácel. Zangemannovo sídlo se nacházelo vysoko na kopci. Celé město měl pod sebou jako na dlani.
```

Comment:
```suggestion
Daleko odtud, na opačném konci města, žil slavný vynálezce Zangemann. Byl nesmírně bohatý. Žádný list papíru na světě nebyl dost velký na to, aby se na něj vešel zůstatek jeho účtu se všemi těmi číslicemi a nulami. Bydlel v obrovském domě s bazénem a tobogánem, mnoha schodišti, věžemi, stovkami oken a tolika pokoji, že i on sám se v nich pravidelně ztrácel. Zangemannovo sídlo se nacházelo vysoko na kopci. Celé město měl pod sebou jako na dlani.
```

Reply:
Nad tím jsem přemýšlel hodně. Představuju si, jak by to mělo vypadat, kdyby text plynul v normální knize bez obrázků. Tam by bylo naprosto jasné, že se mluví o konci města, protože text jde takto:

_... na vrakovišti na konci města. Daleko odtud, na opačném konci, žil slavný ..._

Originální text velmi trpí na opakovaná slova a stylisticky to určitě není Shakespear. Snažme se o co nejlepší čtivost v češtině.

Reply:
Jako jo, pokud by to bylo ve stejné kapitole, tak jo, ale předpokládala jsem, že nový slide je jako nová kapitola, takže najednou chybí kontext. Tady to teda hodně záleží na tom, jak moc jsou oddělené ty slidy.

Reply:
Tak mám za to, že to bude vytištěná kniha, takže bych čekal, že čtenář půjde po tom textu. Počkejme, co řekne @xsuchy.

Reply:
Upřímná otázka. Myslíš, že kdyby to i bylo na dvou slidech, tak bys nebyla ten kontext schopná udržet? Protože já věřím, že ano.

Reply:
Nevím, přijde mi, že je to hodně o tom, jak vypadá ta výsledná knížka, a nemám moc zkušeností se čtením pro děti. Představuji si, že dítěti přečteš kousek, pak se spolu chvilku díváte na obrázek a pak na další stránce pokračujete. A v ten moment ten kontext chybí. Ale nechala bych toto rozhodnutí na někom jiném.

Reply:
Ano, počítal bych s tím, že výsledek bude fyzická kniha. Pokud ji chcete vidět fyzicky stavte se v kanclu kde jsem ji v knihovně nechal. 
Jak to bude vypadat si muzete vyzkouset podle tohoto navodu https://github.com/xsuchy/ada-zangemann-cz/tree/main/scripts#usage
Cilem je IMHO cilovka deti kolem 5-8 roku. A nebál bych se vzít si za vzor Čtvrtka a jeho Rumcajsa, který ty věty sekal extrémně krátké. Takže... co vlastně... asi jsem pro to nebát se opakovat slova. Stejně pak budeme dělat zkušební čtení, tak to kdyžtak upravíme později.